### PR TITLE
Add translations for circleci-images and workflows

### PR DESCRIPTION
**Pull request by ganezasan**
# Description
Add two translations for circleci-images and workflows

# Reasons
https://circleci.atlassian.net/browse/DNAMEDIA-173

### CODE REVIEW

File: jekyll/_cci2_ja/workflows.md [hunk 1]
```diff
@@ -4,68 +4,82 @@ title: "ワークフローを使用したジョブのスケジュール"
...
-
-- 目次
+* 目次
 {:toc}
 
 ## 概要
+{: #overview }
 
-**ワークフロー**は、一連のジョブとその実行順序を定義するルールのセットです。 ワークフローを使用すると、単純な設定キーを組み合わせて複雑なジョブ オーケストレーションを構成でき、問題の早期解決に役立ちます。
+**Workflow** は、ジョブの集まりとその実行順序の定義に関するルールを決めるものです。 ワークフローを使用すると、単純な設定キーを組み合わせて複雑なジョブ オーケストレーションを構成でき、問題の早期解決に役立ちます。
```
Comment (ganezasan) [inline jekyll/_cci2_ja/workflows.md:20]:
```suggestion
**ワークフロー** は、ジョブの集まりとその実行順序の定義に関するルールを決めるものです。 ワークフローを使用すると、単純な設定キーを組み合わせて複雑なジョブ オーケストレーションを構成でき、問題の早期解決に役立ちます。
```

File: jekyll/_cci2_ja/workflows.md [hunk 2]
```diff
@@ -4,68 +4,82 @@ title: "ワークフローを使用したジョブのスケジュール"
...
+* 設定ファイルにワークフローを含めない場合は、`build` という名前のジョブを含める必要があります。
 
-その他の詳細と制限事項については、「[よくあるご質問]({{ site.baseurl }}/ja/2.0/faq)」のワークフロー セクションを参照してください。
+その他の詳細と制限事項については、「よくあるご質問」の「[ワークフロー]({{ site.baseurl }}/ja/2.0/faq/#%E3%83%AF%E3%83%BC%E3%82%AF%E3%83%95%E3%83%AD%E3%83%BC)」セクションを参照してください。
 
 ## ワークフローの構成例
+{: #workflows-configuration-examples }
 
-`workflows` *キーの完全な仕様については、**「CircleCI を設定する」の「[workflows]({{ site.baseurl }}/ja/2.0/configuration-reference/#workflows)」を参照してください。*
+`workflows` _キーの完全な仕様については、__「CircleCI を設定する」の「[workflows]({{ site.baseurl }}/2.0/configuration-reference/#workflows)」セクション_を参照してください。
```
Comment (ganezasan) [inline jekyll/_cci2_ja/workflows.md:60]:
```suggestion
`workflows` *キーの完全な仕様については、**「CircleCI を設定する」**の「[workflows]({{ site.baseurl }}/2.0/configuration-reference/#workflows)」セクション* を参照してください。
```

File: jekyll/_cci2_ja/workflows.md [hunk 3]
```diff
@@ -189,41 +206,50 @@ workflows:
...
+承認後、設定ファイルでの指示に従って残りのワークフローが実行されます。
 
 ## ワークフローのスケジュール実行
+{: #scheduling-a-workflow }
 
-すべてのブランチで、コミットのたびにワークフローを実行するのは、非効率的でコストもかさみます。 代わりに、特定のブランチに対して特定の時刻にワークフローを実行するようにスケジュールを設定できます。 これで、これらのブランチでは、コミットによってジョブがトリガーされなくなります。
+すべてのブランチで、コミットのたびにワークフローを実行するのは、非効率的でコストもかさみます。 代わりに、特定のブランチに対して特定の時刻にワークフローを実行するようにスケジュールを設定できます。 この機能を使った場合は、そのブランチにおけるトリガーとなるジョブからのコミットは無効となります。
 
-リソースに高い負荷がかかるワークフローやレポートを生成するワークフローは、コミットのたびに実行するのではなく、設定ファイルに `triggers` キーを追加してスケジュール実行することを検討してください。 `triggers` キーは、`workflows` キーの下に**のみ**追加されます。 この機能では、指定したブランチに対して、UTC を表す `cron` 構文を使用してワークフローの実行スケジュールを設定できます。
+膨大なリソースを使用する Workflow、あるいは `triggers` キーを利用してコミット時以外にも定期的にリポートを生成するような Workflow を考えてみます。 `triggers` キーは、`workflows` キーの下に**のみ**追加できます。 この機能は、指定したブランチについて、協定世界時 (UTC) を扱う `cron` コマンドの構文で Workflow の実行をスケジューリングできるようにします。
```
Comment (ganezasan) [inline jekyll/_cci2_ja/workflows.md:238]:
```suggestion
膨大なリソースを使用する Workflow、あるいは `triggers` キーを利用してコミット時以外にも定期的にレポートを生成するような Workflow を考えてみます。 `triggers` キーは、`workflows` キーの下に**のみ**追加できます。 この機能は、指定したブランチについて、協定世界時 (UTC) を扱う `cron` コマンドの構文で Workflow の実行をスケジューリングできるようにします。
```

File: jekyll/_cci2_ja/workflows.md [hunk 4]
```diff
@@ -189,41 +206,50 @@ workflows:
...
+**メモ:** ワークフローをスケジュール実行する場合、そのワークフローは個別のユーザー シートとしてカウントされます。
+
+### 夜間実行の例
+{: #nightly-example }
 {:.no_toc}
 
 デフォルトでは、`git push` ごとにワークフローがトリガーされます。 スケジュールに沿ってワークフローをトリガーするには、ワークフローに `triggers` キーを追加し、`schedule` を指定します。
 
-以下の例では、`nightly` ワークフローが毎日午前 0 時 (UTC) に実行されるように構成しています。 `cron` キーは、POSIX `crontab` 構文を使用して指定されます。`cron` 構文の基本については、[crontab の man ページ](https://www.unix.com/man-page/POSIX/1posix/crontab/)を参照してください。 このワークフローは、`master` ブランチと `beta` ブランチで実行されます。
+下記は、`nightly` という Workflow が毎日午前 12 時 00 分 (UTC) に実行されるよう設定した例です。 `cron` は POSIX 規格の `crontab` の構文で表記します。`cron` の書き方については [crontab man page](https://www.unix.com/man-page/POSIX/1posix/crontab/) を参照してください。 この例では、Workflow は `master` と `beta` のブランチにおいてのみ実行されます。
```
Comment (ganezasan) [inline jekyll/_cci2_ja/workflows.md:250]:
```suggestion
下記は、`nightly` という Workflow が毎日午前 0 時 (UTC) に実行されるよう設定した例です。 `cron` は POSIX 規格の `crontab` の構文で表記します。`cron` の書き方については [crontab man page](https://www.unix.com/man-page/POSIX/1posix/crontab/) を参照してください。 この例では、Workflow は `master` と `beta` のブランチにおいてのみ実行されます。
```

File: jekyll/_cci2_ja/workflows.md [hunk 5]
```diff
@@ -500,67 +590,55 @@ workflows:
...
 
 GitHub で [Settings (設定)] > [Branches (ブランチ)] に移動し、保護されているブランチで [Edit (編集)] ボタンをクリックして、設定の選択を解除します (例: https://github.com/your-org/project/settings/branches)。
 
+
 ## 関連項目
+{: #see-also }
 {:.no_toc}
 
-- 1.0 `circle.yml` ファイルから 2.0 `.circleci/config.yml` ファイルへの移行時に、ワークフローを構成に追加する手順については、[1.0 から 2.0 への移行に関するドキュメント]({{ site.baseurl }}/ja/2.0/migrating-from-1-2/)でワークフローの構成手順を参照してください。
+- 1.0 `circle.yml` ファイルから 2.0 `.circleci/config.yml` ファイルへの移行時に、ワークフローを構成に追加する手順については、[1.0 から 2.0 への移行に関するドキュメント]({{ site.baseurl }}/2.0/migrating-from-1-2/)でワークフローの構成手順を参照してください。
```
Comment (ganezasan) [inline jekyll/_cci2_ja/workflows.md:632]:
```suggestion
- 1.0 `circle.yml` ファイルから 2.0 `.circleci/config.yml` ファイルへの移行時に、ワークフローを構成に追加する手順については、[1.0 から 2.0 への移行に関するドキュメント]({{ site.baseurl }}/ja/2.0/migrating-from-1-2/)でワークフローの構成手順を参照してください。
```

File: jekyll/_cci2_ja/workflows.md [hunk 6]
```diff
@@ -500,67 +590,55 @@ workflows:
...
 
-- ワークフローに関するよくある質問と回答については、「[よくあるご質問]({{ site.baseurl }}/ja/2.0/faq)」のワークフロー セクションを参照してください。
+- ワークフローに関するよくある質問と回答については、「[よくあるご質問]({{ site.baseurl }}/2.0/faq)」の「ワークフロー」セクションを参照してください。
```
Comment (ganezasan) [inline jekyll/_cci2_ja/workflows.md:634]:
```suggestion
- ワークフローに関するよくある質問と回答については、「[よくあるご質問]({{ site.baseurl }}/ja/2.0/faq)」の「ワークフロー」セクションを参照してください。
```

File: jekyll/_cci2_ja/circleci-images.md [hunk 7]
```diff
@@ -6,39 +6,86 @@ description: "CircleCI が提供する Docker イメージの一覧"
...
-- 目次
+* 目次
 {:toc}
 
 ## 概要
+{: #overview }
 {:.no_toc}
 
-CircleCI ではすぐに使える Docker イメージを多数提供しています。 一般に、これらのイメージは正式な Docker イメージの拡張版で、特に CI/CD に便利なツールが含まれます。 すべてのビルド済みイメージは、[Docker Hub の CircleCI Org](https://hub.docker.com/search?q=circleci&type=image) から入手できます。 GitHub の `circleci-images` リポジトリには[各 Docker イメージのソース コード](https://github.com/circleci/circleci-images)も用意しています。 これらの [Docker イメージの作成に使用する Dockerfile](https://github.com/circleci-public/circleci-dockerfiles) は `circleci-dockerfiles` リポジトリで確認できます。
+CircleCI ではすぐに使える Docker イメージを多数提供しています。 一般に、これらのイメージは正式な Docker イメージの拡張版で、特に CI/CD に便利なツールが含まれます。 すべてのビルド済みイメージは、[Docker Hub の CircleCI Org](https://hub.docker.com/search?q=circleci&type=image) から入手できます。 GitHub の `circleci-images` リポジトリには、[従来の Docker イメージそれぞれのソース コード](https://github.com/circleci/circleci-images)も用意しています。 各次世代イメージのリポジトリへのリンクについては、[デベロッパー ハブ](https://circleci.com/developer/ja/images/)を参照してください。 [これらの Docker イメージの作成に使用された Dockerfile](https://github.com/circleci-public/circleci-dockerfiles) は、`circleci-dockerfiles` リポジトリで確認できます。
```
Comment (ganezasan) [inline jekyll/_cci2_ja/circleci-images.md:23]:
```suggestion
CircleCI ではすぐに使える Docker イメージを多数提供しています。 一般に、これらのイメージは正式な Docker イメージの拡張版で、特に CI/CD に便利なツールが含まれます。 すべてのビルド済みイメージは、[Docker Hub の CircleCI Org](https://hub.docker.com/search?q=circleci&type=image) から入手できます。 GitHub の `circleci-images` リポジトリには、[従来の Docker イメージそれぞれのソース コード](https://github.com/circleci/circleci-images)も用意しています。 各次世代イメージのリポジトリへのリンクについては、[Developer Hub](https://circleci.com/developer/ja/images/)を参照してください。 [これらの Docker イメージの作成に使用された Dockerfile](https://github.com/circleci-public/circleci-dockerfiles) は、`circleci-dockerfiles` リポジトリで確認できます。
```

File: jekyll/_cci2_ja/circleci-images.md [hunk 8]
```diff
@@ -6,39 +6,86 @@ description: "CircleCI が提供する Docker イメージの一覧"
...
+ビルド済み CircleCI Docker イメージのデモ アプリケーションでの使用例については、[チュートリアル]({{ site.baseurl }}/2.0/tutorials/)を参照してください。
+
+## 次世代コンビニエンス イメージ
+{: #next-generation-convenience-images }
+
+このセクションで紹介する次世代のコンビニエンス イメージは、CI、効率性、確定的動作を念頭に置いてゼロから設計されました。 注目ポイントは次のとおりです。
+
+**スピンアップ時間の短縮** – Docker 的な言い方をすれば、次世代イメージは概してレイヤーがより少なく、より小さくなっています。 これらの新しいイメージを使用すると、ビルド開始時にイメージがすばやくダウンロードされると共に、イメージが既にホストにキャッシュされている可能性が高くなります。
+
+**信頼性と安定性の向上** – 従来のイメージは、アップストリームからの変更によってほぼ毎日再ビルドされるため、テストが間に合わないこともあります。 そのため、互換性の損なわれる変更が頻発してしまい、安定した確定的なビルドに最適な環境とは言えなくなっています。 次世代イメージは、セキュリティと致命的なバグについてのみ再ビルドされるため、より安定した確定的なイメージとなります。
```
Comment (ganezasan) [inline jekyll/_cci2_ja/circleci-images.md:39]:
```suggestion
**信頼性と安定性の向上** – 従来のイメージは、アップストリームからの変更によってほぼ毎日再ビルドされるため、テストが十分に行われていない場合があります。 そのため、互換性の損なわれる変更が頻発してしまい、安定した確定的なビルドに最適な環境とは言えなくなっています。 次世代イメージは、セキュリティと致命的なバグについてのみ再ビルドされるため、より安定した確定的なイメージとなります。
```

File: jekyll/_cci2_ja/circleci-images.md [hunk 9]
```diff
@@ -6,39 +6,86 @@ description: "CircleCI が提供する Docker イメージの一覧"
...
+
+これは必要最低限のものをインストールするように設計された、まったく新しい Ubuntu ベースのイメージです。 今後数週間でリリースする予定の次世代コンビニエンス イメージはすべてこれがベースとなります。
+
+**最適な用途**
+
+汎用的なイメージを CircleCI で実行したり、Orbs で使用したり、独自のカスタム Docker イメージのベースとして利用したりする必要がある場合に、このイメージをお使いください。
+
+**関連資料**
+
+このイメージの設定ファイルのサンプルは[デベロッパー ハブ](https://circleci.com/developer/ja/images/image/cimg/base)、ソース コードとドキュメントは [GitHub](https://github.com/CircleCI-Public/cimg-base) で入手できます。
```
Comment (ganezasan) [inline jekyll/_cci2_ja/circleci-images.md:56]:
```suggestion
このイメージの設定ファイルのサンプルは[Developer Hub](https://circleci.com/developer/ja/images/image/cimg/base)、ソース コードとドキュメントは [GitHub](https://github.com/CircleCI-Public/cimg-base) で入手できます。
```

File: jekyll/_cci2_ja/circleci-images.md [hunk 10]
```diff
@@ -6,39 +6,86 @@ description: "CircleCI が提供する Docker イメージの一覧"
...
+## 次世代 CircleCI イメージ
+{: #next-gen-circleci-images }
+
+CircleCI では新しいイメージをまとめたリポジトリへの移行を進めています。このイメージは、より詳細なドキュメントを備え、動作の確定性が向上したものです。 以下に、次世代 Go イメージの定義例を示します。
+
+```yaml
+image: cimg/go:1.16
+```
+
+これは従来の CircleCI Go イメージ (`circleci/golang`) の直系の後継となるものです。 Docker Hub の名前空間は `cimg` であることに留意してください。
```
Comment (ganezasan) [inline jekyll/_cci2_ja/circleci-images.md:67]:
```suggestion
これは従来の CircleCI Go イメージ (`circleci/golang`) の後継となるものです。 Docker Hub の名前空間は `cimg` であることに留意してください。
```

File: jekyll/_cci2_ja/circleci-images.md [hunk 11]
```diff
@@ -118,25 +175,53 @@ CircleCI は、以下の言語に対応するイメージを提供していま
...
+
+上記以外の言語のイメージが必要な場合は、CircleCI の[アイデア ボード](https://ideas.circleci.com/)にリクエストしてください。 ただし、リクエストの前に、アイデア ボード内を検索し、 同じアイデアがすでに投稿されている場合は、そのアイデアに投票してください。 まだ投稿されていなければ、カテゴリを [New Image Request (新しいイメージのリクエスト)] に設定してアイデアを投稿してください。 その後、アイデアを友人や同僚、フォーラム、その他のコミュニティに紹介して、票を集めることをお勧めします。
+
+CircleCI では、獲得票数の多いアイデアほど、正式な開発を優先的に検討しています。
+
+#### 次世代言語イメージのバリアント
+{: #next-gen-language-image-variants }
+{:.no_toc}
+
+CircleCI は、次世代言語イメージに対していくつかのバリアントを用意しています。 次世代イメージについては、それぞれのバリアントをイメージごとに確認するようにしてください。 次世代イメージの `-browsers` バリアントは現在作成中です。 サポートされているバリアントの詳細については、[デベロッパー ハブ](https://circleci.com/developer/ja/images)でイメージの一覧を参照してください。
```
Comment (ganezasan) [inline jekyll/_cci2_ja/circleci-images.md:216]:
```suggestion
CircleCI は、次世代言語イメージに対していくつかのバリアントを用意しています。 次世代イメージについては、それぞれのバリアントをイメージごとに確認するようにしてください。 次世代イメージの `-browsers` バリアントは現在作成中です。 サポートされているバリアントの詳細については、[Developer Hub](https://circleci.com/developer/ja/images)でイメージの一覧を参照してください。
```

File: jekyll/_cci2_ja/circleci-images.md [hunk 12]
```diff
@@ -147,13 +232,21 @@ CircleCI は、以下のサービスに対応するイメージを提供して
...
+CircleCI は、サービスイメージに対してバリアント 1つのみ用意しています。 RAM ボリュームを使用してビルドを高速化するには、サービスイメージタグの末尾に `-ram` サフィックスを追加します。
+
+参考までに、`circleci/postgres:9.5-postgis` イメージで RAM ディスクを使うには、`circleci/postgres:9.5-postgis-ram` とします。
+
+### 次世代サービス イメージ
+{: #next-gen-service-images }
+{:.no_toc}
 
-たとえば、`circleci/postgres:9.5-postgis` イメージで RAM ボリュームを使用する場合は、`circleci/postgres:9.5-postgis-ram` イメージを使用します。
+CircleCI では、次世代サービス コンビニエンス イメージの拡充に取り組んでいます。 使用可能な最新のサービス イメージについては、CircleCI の[デベロッパー ハブ](https://circleci.com/developer/ja/images/)を参照してください。
```
Comment (ganezasan) [inline jekyll/_cci2_ja/circleci-images.md:246]:
```suggestion
CircleCI では、次世代サービス コンビニエンス イメージの拡充に取り組んでいます。 使用可能な最新のサービス イメージについては、CircleCI の[Developer Hub](https://circleci.com/developer/ja/images/)を参照してください。
```

**Review by ogii — APPROVED** [review]

**Review by kurumai — APPROVED** [review]
LGTM

File: jekyll/_cci2_ja/circleci-images.md [hunk 13]
```diff
@@ -61,50 +111,57 @@ jobs:
...
 {:.no_toc}
 
-[イメージ タグ](https://docs.docker.com/engine/reference/commandline/tag/#extended-description)を追加することで、Docker イメージの状態を固定できます。
+[イメージタグ](https://docs.docker.com/engine/reference/commandline/tag/#extended-description)を利用することで Docker イメージの役割を決めることができます。
 
-たとえば、`circleci/golang` の代わりに `circleci/golang:1.8.6-jessie` を使用して、バージョンと OS を指定します。 後者はバージョンと OS を指定しているため、意図しない変更の影響を受けるリスクが抑えられます。
+たとえば、`cimg/go:1.16` の代わりに `cimg/go:1.16.3` として、バージョンを指定します。 後者は特定のバージョンを指定しているため、意図しない変更の影響を受けるリスクが抑えられます。
 
-以降に掲載した[言語別の最新イメージ タグ](#言語別の最新イメージ-タグ)の一覧を参照してください。
+以降に掲載した[言語別の最新イメージ タグ](#%E8%A8%80%E8%Aa%9E%E5%88%A5%E3%81%Ae%E6%9C%80%E6%96%B0%E3%82%A4%E3%83%A1%E3%83%Bc%E3%82%B8-%E3%82%Bf%E3%82%B0)の一覧を参照してください。
```
Comment (teka23) [inline jekyll/_cci2_ja/circleci-images.md:122]:
```suggestion
以降に掲載した[言語別の最新イメージ タグ](#latest-image-tags-by-language)の一覧を参照してください。
```

File: jekyll/_cci2_ja/circleci-images.md [hunk 14]
```diff
@@ -118,25 +175,53 @@ CircleCI は、以下の言語に対応するイメージを提供していま
...
 - `-node-browsers`: `-node` バリアントと `-browsers` バリアントの組み合わせです。
 
-たとえば、`circleci/golang:1.9` イメージにブラウザーを追加する場合は、`circleci/golang:1.9-browsers` イメージを使用します。
+参考までに、`circleci/golang:1.9` に Web ブラウザ をインストールしておきたいときは `circleci/golang:1.9-browsers` とします。
+
+### 次世代言語イメージ
+{: #next-gen-language-images }
+{:.no_toc}
+
+次世代言語イメージは、従来のものと同様、一般的なプログラミング言語に対応するコンビニエンス イメージであり、 関連する言語と[共通して使用されるツール](#%E3%83%97%E3%83%AA%E3%82%A4%E3%83%B3%E3%82%B9%E3%83%88%E3%83%BC%E3%83%AB-%E3%83%84%E3%83%BC%E3%83%AB)の両方が含まれます。 言語イメージを指定するときは、設定ファイル内の `docker` キー配下の最初の行に挿入します。したがって、ビルドの実行中はこれが[プライマリコンテナ]({{ site.baseurl }}/2.0/glossary/#primary-container){:target="_blank"}になります。
```
Comment (teka23) [inline jekyll/_cci2_ja/circleci-images.md:194]:
```suggestion
次世代言語イメージは、従来のものと同様、一般的なプログラミング言語に対応するコンビニエンス イメージであり、 関連する言語と[共通して使用されるツール](#pre-installed-tools)の両方が含まれます。 言語イメージを指定するときは、設定ファイル内の `docker` キー配下の最初の行に挿入します。したがって、ビルドの実行中はこれが[プライマリコンテナ]({{ site.baseurl }}/2.0/glossary/#primary-container){:target="_blank"}になります。
```

File: jekyll/_cci2_ja/circleci-images.md (hunk 13 above)
Comment (ganezasan) [reply jekyll/_cci2_ja/circleci-images.md:122]: なるほど！

File: jekyll/_cci2_ja/circleci-images.md [hunk 15]
```diff
@@ -173,72 +266,81 @@ CircleCI は、サービス イメージに対してバリアント 1 つのみ
...
+**従来の**コンビニエンス イメージについて、最新のものを言語別に紹介します。 それぞれの詳細については [corresponding Dockerfiles](https://github.com/circleci-public/circleci-dockerfiles) でご確認ください。
 
-最新のコンビニエンス イメージを言語別に紹介します。 各イメージの内容の詳細については、[対応する Dockerfile](https://github.com/circleci-public/circleci-dockerfiles) を参照してください。
 
-**メモ:** CircleCI は、[言語イメージのバリアント](#言語イメージのバリアント)および[サービス イメージのバリアント](#サービス-イメージのバリアント)以外のイメージでどのようなタグが使用されるかを**関知しません**。 これらのタグは、アップストリーム プロジェクトによって選択および維持されます。 特定のタグがイメージ間で同じ意味を持つと断定しないように注意してください。
+<div class="alert alert-warning" role="alert">
+最新の次世代コンビニエンス イメージの一覧と各イメージの内容の詳細については、<a href="https://circleci.com/developer/ja/">デベロッパー ハブ</a>を参照してください。
+</div>
+
+**注:** CircleCI は、[言語イメージのバリアント](#%E8%A8%80%E8%Aa%9E%E3%82%A4%E3%83%A1%E3%83%Bc%E3%82%B8%E3%81%Ae%E3%83%90%E3%83%Aa%E3%82%A2%E3%83%B3%E3%83%88)および[サービス イメージのバリアント](#%E3%82%B5%E3%83%Bc%E3%83%93%E3%82%B9-%E3%82%A4%E3%83%A1%E3%83%Bc%E3%82%B8%E3%81%Ae%E3%83%90%E3%83%Aa%E3%82%A2%E3%83%B3%E3%83%88)以外の**従来のイメージ**で、どのようなタグが使用されるかを**関知しません**。 これらのタグは開発元が手がけるプロジェクトとして作成、メンテナンスされています。 似た名前のタグでも同じような内容のイメージとは限らないことにご注意ください。
```
Comment (teka23) [inline jekyll/_cci2_ja/circleci-images.md:300]:
```suggestion
**注:** CircleCI は、[言語イメージのバリアント](#language-image-variants)および[サービス イメージのバリアント](#service-image-variant)以外の**従来のイメージ**で、どのようなタグが使用されるかを**関知しません**。 これらのタグは開発元が手がけるプロジェクトとして作成、メンテナンスされています。 似た名前のタグでも同じような内容のイメージとは限らないことにご注意ください。
```

Comment (ganezasan) [inline jekyll/_cci2_ja/circleci-images.md:300]:
```suggestion
**注:** CircleCI は、[言語イメージのバリアント](#language-image-variants)および[サービス イメージのバリアント](#service-image-variant)以外の**従来のイメージ**で、どのようなタグが使用されるかを**関知しません**。 これらのタグは開発元が手がけるプロジェクトとして作成、メンテナンスされています。 似た名前のタグでも同じような内容のイメージとは限らないことにご注意ください。
```

**Review by teka23 — APPROVED** [review]
LGTM with some suggestions

File: jekyll/_cci2_ja/workflows.md [hunk 16]
```diff
@@ -324,19 +371,20 @@ workflows:
...
+正規表現の詳細については、この後の「[正規表現を使用してタグとブランチをフィルタリングする](#%E6%AD%A3%E8%A6%8F%E8%A1%A8%E7%8F%BE%E3%82%92%E4%BD%BF%E7%94%A8%E3%81%97%E3%81%A6%E3%82%BF%E3%82%B0%E3%81%A8%E3%83%96%E3%83%A9%E3%83%B3%E3%83%81%E3%82%92%E3%83%95%E3%82%A3%E3%83%AB%E3%82%BF%E3%83%AA%E3%83%B3%E3%82%B0%E3%81%99%E3%82%8B)」を参照してください。
 
 ワークフロー構成例の全文は、ブランチを含む順次ワークフロー サンプル プロジェクトの[設定ファイル](https://github.com/CircleCI-Public/circleci-demo-workflows/blob/sequential-branch-filter/.circleci/config.yml)でご覧いただけます。
 
 ### Git タグに対応するワークフローを実行する
+{: #executing-workflows-for-a-git-tag }
 {:.no_toc}
 
-明示的にタグ フィルターを設定しない限り、CircleCI はタグに関するワークフローを実行しません。 さらに、ジョブが (直接的または間接的に) 他のジョブを必要とする場合は、[正規表現を使用](#正規表現を使用してタグとブランチをフィルタリングする)して、それらのジョブに対応するタグ フィルターを指定する必要があります。 軽量のタグと注釈付きのタグがサポートされています。
+CircleCI は明示的にタグフィルターを指定しない限り、タグに対して Workflows は実行しません。 さらに、ジョブが (直接的または間接的に) 他のジョブを必要とする場合は、[正規表現を使用](#%E6%AD%A3%E8%A6%8F%E8%A1%A8%E7%8F%BE%E3%82%92%E4%BD%BF%E7%94%A8%E3%81%97%E3%81%A6%E3%82%BF%E3%82%B0%E3%81%A8%E3%83%96%E3%83%A9%E3%83%B3%E3%83%81%E3%82%92%E3%83%95%E3%82%A3%E3%83%AB%E3%82%BF%E3%83%AA%E3%83%B3%E3%82%B0%E3%81%99%E3%82%8B)して、それらのジョブに対応するタグ フィルターを指定する必要があります。 CircleCI では軽量版と注釈付き版のどちらのタグにも対応しています。
```
Comment (ganezasan) [inline jekyll/_cci2_ja/workflows.md:382]:
```suggestion
CircleCI は明示的にタグフィルターを指定しない限り、タグに対して Workflows は実行しません。 さらに、ジョブが (直接的または間接的に) 他のジョブを必要とする場合は、[正規表現を使用](#using-regular-expressions-to-filter-tags-and-branches)して、それらのジョブに対応するタグ フィルターを指定する必要があります。 CircleCI では軽量版と注釈付き版のどちらのタグにも対応しています。
```

File: jekyll/_cci2_ja/workflows.md [hunk 17]
```diff
@@ -324,19 +371,20 @@ workflows:
               only: /pre-prod(?:-.+)?$/
 ```
 
-正規表現の詳細については、この後の「[正規表現を使用してタグとブランチをフィルタリングする](#正規表現を使用してタグとブランチをフィルタリングする)」を参照してください。
+正規表現の詳細については、この後の「[正規表現を使用してタグとブランチをフィルタリングする](#%E6%AD%A3%E8%A6%8F%E8%A1%A8%E7%8F%BE%E3%82%92%E4%BD%BF%E7%94%A8%E3%81%97%E3%81%A6%E3%82%BF%E3%82%B0%E3%81%A8%E3%83%96%E3%83%A9%E3%83%B3%E3%83%81%E3%82%92%E3%83%95%E3%82%A3%E3%83%AB%E3%82%BF%E3%83%AA%E3%83%B3%E3%82%B0%E3%81%99%E3%82%8B)」を参照してください。
```
Comment (ganezasan) [inline jekyll/_cci2_ja/workflows.md:374]:
```suggestion
正規表現の詳細については、この後の「[正規表現を使用してタグとブランチをフィルタリングする](#using-regular-expressions-to-filter-tags-and-branches)」を参照してください。
```

**Pull request merged**